### DISCUSSION TITLE
Trouble Installing mason-lspconfig.nvim

### DISCUSSION THREAD

**Discussion by quinnodel** [mason-org/mason.nvim#1494] [Q&A]
Hi there! 

I am having trouble installing mason-lspconfig. 

The documentation has install instructions with Packer and Vim Plug, but not with Lazy. Are there any instructions on how to download this with Lazy? I have done as much research as I can, but configuring plugins with Lua is very different than with Vim Plug.

Any help would be appreciated, I really just want to get LSP working properly and didn't realize it was going to be this much of a run-around. Thank you!

**Comment by williamboman** [comment]
Hello, see https://www.lazyvim.org/configuration/plugins#-adding-plugins
